pandas/core/groupby/generic.py

```diff
@@ -518,6 +548,14 @@ def filter(self, func, dropna: bool = True, *args, **kwargs):
         3    4
         5    6
         Name: B, dtype: int64
+        >>> grouped.transform("mean")
```

Comment:
Looks like this was added to the wrong function; this is the docstring of `filter`.

Reply:
Ohh, great catch, I apologize I didn't notice that earlier. This has been resolved here: 9223b59e6aac77dc0a12eb8585ce1e8e0c9f52d4